### ISSUE TITLE
Naming of method Value::make_global_ref

### ISSUE THREAD

**Issue by cireu** [ubolonton/emacs-module-rs#32]
According to https://rust-lang.github.io/api-guidelines/naming.html. I think we'd better to rename `make_global_ref` to `to_global_ref` to help user discover this API. What's your opinion?

**Comment by cireu** [comment]
Or even `into_global_ref`? though `Value` derives `Copy`, but it's actually a proxy type of Emacs value, not a POD in true sense.

**Comment by ubolonton** [comment]
Yeah, either `to_` or `into_` sounds better. I'm leaning towards the former, but I'm not sure yet.
